packer/ansible/roles/node/tasks/main.yml

```diff
@@ -1,15 +1,21 @@
 ---
-- name: Download n Installer
-  get_url: url=http://git.io/n-install dest=/tmp/n-install mode=0755
-  register: download_n
-  sudo: yes
 
-- name: Install n
-  shell: "yes | N_PREFIX={{ ansible_env.HOME }}/n /tmp/n-install"
-  sudo: yes
-  when: download_n|changed
+# source ~/.bashrc is for interactive shell
+- name: Install nvm and Appending source string to ~/.bashrc
+  shell: wget -qO- https://raw.githubusercontent.com/creationix/nvm/v0.31.0/install.sh | bash
```

Comment:
I think we're going to be much better off installing `n` rather than `nvm`, the reason being that `nvm` is a real pain to use when you need to install the same node version for the root and vagrant user, whereas with `n` you just do `sudo n <version>` and it automatically gets installed for all users.